### DISCUSSION TITLE
Classic Boards

### DISCUSSION THREAD

**Discussion by BobbyPr** [missionpinball/missionpinball#102] [Q&A]
Is there anyway to use MPF to program some of the old classic pinball games like 8 Ball (by Bally), Joker Poker (by Gottlieb) etc...
are the ROMS, used with Visual Pinball, for those games of any use in MPF? 
When I look at the scripts for those games they don't even look close to MPF
Any help / resources / direction would be greatly appreciated

**Comment by atummons** [comment]
They are certainly not going to be portable. But if they are good and complete, it will tell you all the logic, timing, sequence of shots, modes, etc. It is likely better than nothing, but will take some conversion effort.

**Reply by BobbyPr** [reply]
i found how MPF can control an _existing_ game. It's just replacing the existing control board, but i would like to actually build the box and use  use monitors for my display boards. doesn't sound like mpf can do this
